code/datums/spells/knock.dm

```diff
@@ -13,6 +13,11 @@
 	action_icon_state = "knock"
 	sound = 'sound/magic/knock.ogg'
 
+// Knock doesn't need to generate an attack log for every turf, set `make_attack_logs` to FALSE and just create a custom one.
+/obj/effect/proc_holder/spell/aoe_turf/knock/perform(list/targets, recharge, mob/user)
+	add_attack_logs(user, user, "cast the spell [name]", ATKLOG_ALL)
```

Comment:
Can give the target here as target for the logs. That way admins can jump to the location if need be

Reply:
Well see, knock basically passes in a screen's worth of turfs into the `targets` list, which is why I didn't want to do that.